pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/gardeneruser/component.go

```diff
@@ -46,8 +47,8 @@ func init() {
 }
 
 const (
-	pathScript       = "/var/lib/gardener-user/run.sh"
-	pathPublicSSHKey = "/var/lib/gardener-user-ssh.key"
```

Comment:
Should we pro-actively delete this file on existing nodes? Otherwise, it will stay there forever until the node is replaced. Probably it doesn't harm and it could be acceptable to prevent additional code, though, WDYT?

Reply:
Yeah let's just delete the leftovers.